go/runtime/registry/host_rofl_mgmt.go

```diff
@@ -68,21 +91,44 @@ func (rh *roflHostHandler) handleBundleWrite(rq *rofl.BundleWriteRequest) (*rofl
 	return &rofl.BundleWriteResponse{}, nil
 }
 
-func (rh *roflHostHandler) handleBundleAdd(rq *rofl.BundleAddRequest) (*rofl.BundleAddResponse, error) {
+func (rh *roflHostHandler) handleBundleAdd(ctx context.Context, rq *rofl.BundleAddRequest) (*rofl.BundleAddResponse, error) {
 	if err := rh.ensureComponentPermissions(runtimeConfig.PermissionBundleAdd); err != nil {
 		return nil, err
 	}
 
+	select {
+	case <-rh.getVolumeManager().Initialized():
```

Comment:
Ideally, this shouldn't be needed. Yes, the manager needs to run to load volumes and clean them, but if it is not running it would be still nice if we could call its methods. 
For example, on the first request you could load all volumes and register them, if they haven't been so far. Or, like is done in the bundle manager, you first create a volume, and if meanwhile `run` registered it, you will get an error.

Reply:
Ups, the second option is not possible because names are random and creation of a volume will always succeed, even if one with the same labels already exists locally.

Reply:
Will change to use the first variant.

Reply:
Changed, please take a look.